### PR TITLE
Fix processing address from qr code on cancel send

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

**Comment by mateuszpmroz** [comment]
Also in `src/screens/PinFlow/CreateTransactionPassword.tsx` could be the problem with height and visibility icon. Height for container could be set up there aswell.

File: src/screens/PinFlow/ConfirmTransactionPassword.tsx [hunk 1]
```diff
@@ -134,6 +134,7 @@ const styles = StyleSheet.create({
   inputItemContainer: {
     paddingTop: 20,
     width: '100%',
+    height: 100,
```
Comment (marek-siemieniuk-morawski) [inline src/screens/PinFlow/ConfirmTransactionPassword.tsx:137]: what does mean `height: 100` without any unit? How does it work?

**Review by marek-siemieniuk-morawski — APPROVED** [review]

File: src/screens/PinFlow/ConfirmTransactionPassword.tsx (hunk 1 above)
Comment (KedziaPawel) [reply src/screens/PinFlow/ConfirmTransactionPassword.tsx:137]: px, a constant value of height to prevent jumping this icon visibility

Comment (marek-siemieniuk-morawski) [reply src/screens/PinFlow/ConfirmTransactionPassword.tsx:137]: k, thanks 😊

**Comment by KedziaPawel** [comment]
> Also in `src/screens/PinFlow/CreateTransactionPassword.tsx` could be the problem with height and visibility icon. Height for container could be set up there aswell.

added

**Pull request merged**